metricbeat/etc/beat.yml

```diff
@@ -1,85 +1,87 @@
-metricbeat.modules:
+###################### Metricbeat Configuration Example #######################
 
-  # Apache Module
-  - module: apache
-    metricsets: ["status"]
-    enabled: true
-    period: 1s
+#==========================  Modules configuration ============================
+metricbeat.modules:
 
-    # Apache hosts
-    hosts: ["http://127.0.0.1/"]
+#----------------------------- Apache Module ----------------------------------
```

Comment:
As this is auto generated, it will not last long ;-) Has to be adjusted in the script. I can do that for you after merging if you like.